### DISCUSSION TITLE
Halting New Ruleset Additions May 31st, 2021

### DISCUSSION THREAD

**Discussion by zoracon** [EFForg/https-everywhere#20099] [Announcements]
We will be phasing out our rulesets in favor of DuckDuckGo Smarter Encryption List that we have in an update channel.
---

The phases are as listed
-   DuckDuckGo Update Channel with Smarter Encryption Rulesets [April 15, 2021].
-   Still accept HTTPS Everywhere Ruleset changes in Github Repo until the end of May, 2021.
-   Still host HTTPS Everywhere Rulesets until various partners and downstream channels that use our current rulesets, make needed changes and decisions.
-   Sunset HTTPS Everywhere Rulesets [Late 2021]

We will still accept requests on rulesets already in our list that are causing significant breakage for users through the summer. However in autumn, we will begin the plan to ultimately move our crowdsourced rulesets out of the extension.

Thank you again to all of our maintainers who diligently have been working on HTTPS Everywhere rulesets all these years.

**Comment by J0WI** [comment]
IMHO the Smarter Encryption list is currently not smarter than HTTPS Everywhere rulesets.
While it contains more hosts we may loose significant coverage for specific sites.

I'm curious if there are ways to improve: https://github.com/duckduckgo/smarter-encryption/issues

**Reply by zoracon** [reply]
I agree that our rulesets are more filtered and finely tuned for many different cases. Our approach was crowd sourced knowledge and deeply intimate knowledge of domains usually. I would have been more worried about this aspect if the browser's weren't building in HTTPS only capabilities. I plan to keep our list running through this year but considering most web traffic in browser is above 95%, I think we can say the many different cases we accounted for won't be as impactful. 

However, I get that there are some desired traits for Smarter Encryption and I can pass those concerns along to the DDG team.
